pyproject.toml

```diff
@@ -16,10 +16,9 @@ files = "src/guidellm"
 [tool.ruff]
 exclude = ["build", "dist", "env", ".venv", "src/llmcompressor/transformers/tracing/"]
 lint.select = ["E", "F", "W"]
+lint.extend-ignore = ["E203", "W605"]
 
-[tool.flake8]
-max-line-length = 88
-extend-ignore = 'E203'
+# for flake8 configuration, see `Makefile`
 
```

Comment:
Why the changes here? Having redundancy in `pyproject.toml` is valuable for things not using Makefile, like VSCode's format on save, right?